packages/cdktf/lib/terraform-iterator.ts

```diff
@@ -80,6 +81,26 @@ export abstract class TerraformIterator implements ITerraformIterator {
     return new MapTerraformIterator(map);
   }
 
+  /**
+   * Creates a new iterator from a resource that
+   * has been created with the `for_each` argument.
+   */
+  public static fromResources(
+    resource: ITerraformResource
+  ): ResourceTerraformIterator {
+    return new ResourceTerraformIterator(resource);
+  }
+
+  /**
+   * Creates a new iterator from a data source that
+   * has been created with the `for_each` argument.
+   */
+  public static fromDataSources(
+    resource: ITerraformResource
```

Comment:
Is there a specific type for DataSources? Or is the method just doubled for keeping the name simple?

Reply:
It's just an alias so that we don't get people confused on if it's only supported for Resources :)